ikvmc/Compiler.cs

```diff
@@ -952,6 +969,26 @@ private static int ResolveReferences(List<CompilerOptions> targets)
 				next_reference: ;
 				}
 			}
+			if (target.unresolvedNetStandardReferences != null)
+			{
+				foreach (string reference in target.unresolvedNetStandardReferences)
+				{
+					foreach (CompilerOptions peer in targets)
+					{
+						if (peer.assembly.Equals(reference, StringComparison.InvariantCultureIgnoreCase))
+						{
+							ArrayAppend(ref target.peerReferences, peer.assembly);
+							goto next_reference;
+						}
+					}
+					int rc = resolver.ResolveReference(cache, ref target.references, reference, true);
+					if (rc != 0)
+					{
+						return rc;
```

Comment:
I don't like this return statement. Not all peer assemblies could be added to `target.peerReferences` sometime.